File: src/main/java/com/cloudpi/cloudpi_backend/files/filesystem/repositories/FileIdRepository.java
package com.cloudpi.cloudpi_backend.files.filesystem.repositories;

import com.cloudpi.cloudpi_backend.files.filesystem.entities.FilesystemIdEntity;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
interface FileIdRepository extends JpaRepository<FilesystemIdEntity, Long> {

}

File: src/main/java/com/cloudpi/cloudpi_backend/files_info/repositories/FileIdRepository.kt
package com.cloudpi.cloudpi_backend.files_info.repositories

import com.cloudpi.cloudpi_backend.files_info.entities.kotlin.FilesystemIdEntity
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface FileIdRepository : JpaRepository<FilesystemIdEntity, Long>